p5js/Pix2Pix_callback/sketch.js

```diff
@@ -61,13 +67,17 @@ function transfer() {
 
   // Select canvas DOM element
   let canvasElement = document.getElementById('defaultCanvas0');
+
   // Apply pix2pix transformation
-  pix2pix.transfer(canvasElement, function(result) {
-    // Clear output container
-    outputContainer.html('');
-    // Create an image based result
-    createImg(result.src).class('border-box').parent('output');
+  pix2pix.transfer(canvasElement, function(err, result) {
+    if (err) console.log(err);
```

Comment:
For p5 examples it's generally preferable to always use `{}` even when they can be left out.